### PR TITLE
Feature/list

### PR DESCRIPTION
**Pull request by ethan-qsh**

### CODE REVIEW

File: libs/react/src/lib/list/listItem.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  listType?: ListType
+  children?: ReactNode
+}
+
+const ListItem = ({ listType, children, ...props }: ListItemProps) => {
+  if (listType === 'table') {
+    return <div>{children}</div>
+  } else {
+    return <li {...props}>{children}</li>
+  }
```
Comment (mario-subo) [inline libs/react/src/lib/list/listItem.tsx:14]: I guess you will have to add another condition to wrap the children in `<dl>` as well

**Comment by mario-subo** [comment]
Note: I've added re #314 and re #315 which relate to the list but the style will probably have to be fixed in chlorophyll

File: libs/react/src/lib/list/listItem.tsx (hunk 1 above)
Comment (ethan-qsh) [reply libs/react/src/lib/list/listItem.tsx:14]: Yeap, will have to look into how to implement that. The `<div>` on line 11 is used to style the table that's why its there

Comment (ethan-qsh) [reply libs/react/src/lib/list/listItem.tsx]: I made the changes and I saw that the tag was changed from `<dl>` to `<dd>` in another PR so I made the changes accordingly

**Review by hjalmers — APPROVED** [review]

**Pull request merged**